### PR TITLE
Fix merging

### PR DESCRIPTION
**Pull request by wuallenwu**
Merging is now fixed up to but not including inertial stuff! The amount of contact forces is much smaller, only two infinite contacts remain, need to diagnose what that is.

### CODE REVIEW

**Comment by wuallenwu** [comment]
To get the merged urdf

```
kol urdf https://cad.onshape.com/documents/afaee604f6ca311526a6aec8/w/29af84cb974c2d825b71de39/e/4fef6bce7179a665e62b03ba 
&& kol merge-fixed-joints robot/assembly_1.urdf 
&& kol simplify-all robot/assembly_1_merged.urdf
```

The simplified and merged urdf will be output as robot/assembly_1_merged_simplified.urdf.

Additionally, the option to cleanup_fused_meshes after merging works now, it successfully removes all of the meshes not mentioned in the meshes folder.

workingmergedsimplified

**Comment by hu-po** [comment]
send it

**Review by hu-po — APPROVED** [review]

File: kol/simplify_all.py [hunk 1]
```diff
@@ -6,12 +6,20 @@
...
 
+total_removed = 0
+
 
 def simplify_mesh(filepath: str, voxel_size: float) -> tuple[str, str]:
     """Simplifies a mesh by clustering vertices."""
+    # if file path doesn't include /meshes/, add it in after robot/
+    if "/meshes/" not in filepath:
+        filepath = filepath.replace("robot/", "robot/meshes/")
+    print(filepath)
```
Comment (budzianowski) [inline kol/simplify_all.py:22]: remove

**Review by budzianowski — COMMENTED** [review]
Can you add explanation what is the main change - after reading the PR I'm still not sure? If the merging works, can you explain why inertia is omitted?

**Pull request merged**